config/routes.rb

```diff
@@ -4,6 +4,10 @@
   # Defines the root path route ("/")
   root "users#index"
   get '/user/:id', to: 'users#show'
+  get '/post/new', to: 'posts#new'
+  post '/post/new', to: 'posts#create'
+  post '/addcomment/:id', to: "posts#addcomment"
   get '/user/:id/post/', to: 'posts#posts'
   get '/user/:id/post/:id', to: 'posts#post'
+  get 'addlike/:id', to: "posts#addlike"
```

Comment:
- [ ] After including the comments and likes controller, Also update the routes.